packages/storybook/src/nimble/patterns/tabs/doc-strings.ts

```diff
@@ -0,0 +1,4 @@
+import { tabsToolbarTag } from '@ni/nimble-components/dist/esm/tabs-toolbar';
+
+export const defaultSlotDescription = `Slot to display elements on the left side of the \`${tabsToolbarTag}\`. Elements that do not specify a slot will be placed in this slot.`;
+export const endSlotDescription = `Slot to display elements on the right side of the \`${tabsToolbarTag}\`. If slotted content is only in the \`end\` slot, the separator will be hidden.`;
```

Comment:
nit: I don't think we need to explain how default slot works here. And I think it's clearer to talk about the separator on the slot that actually controls it. (I'd also be ok to not even mention the separator, it's kind of a styling implementation detail)

```suggestion
export const defaultSlotDescription = `Slot to display elements on the left side of the \`${tabsToolbarTag}\`. If this slot is empty, the separator will be hidden.`;`;
export const endSlotDescription = `Slot to display elements on the right side of the \`${tabsToolbarTag}\`.
```